frontend/src/tests/components/eMIB/ActionViewTask.test.js

```diff
@@ -87,3 +87,15 @@ describe("check that the disabled prop works as expected", () => {
     expect(wrapper.find("#unit-test-view-task-delete-button").exists()).toEqual(false);
   });
 });
+
+it("it renders when the email is undefined", () => {
+  shallow(
+    <UnconnectedActionViewTask
+      actionId={0}
+      action={{ actionType: ACTION_TYPE.task }}
+      email={emailStub}
```

Comment:
and email is defined here.

Reply:
as mentioned offline, "email" here is the original email that is being replied to; "action" is the response, which always has a type assigned.